setup.py

```diff
@@ -29,6 +29,9 @@
         "recommonmark==0.7.1",
         "sphinx_rtd_theme==0.5.1",
     ],
+    "gcs": [
```

Comment:
Let's change it to `gcp`, so it's more related to environment, less to specific service.

Reply:
Typo, already changed.